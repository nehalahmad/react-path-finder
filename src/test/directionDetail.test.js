import React from 'react';
import { shallow } from 'enzyme';

import DirectionDetail from '../components/form/directionDetail';

describe("<DirectionDetail />", () => {
    let direction = {
        total_distance: 1000,
        total_time: 500
    };
    it("direction detail is rendered", () => {
        const DirectionDetailComp = shallow(<DirectionDetail direction={direction} />);
        expect(DirectionDetailComp).toMatchSnapshot();
    });
    it("direction detail is rendered", () => {
        const DirectionDetailComp = shallow(<DirectionDetail message="This location is not accessible by Car" />);
        expect(DirectionDetailComp).toMatchSnapshot();
    });
});