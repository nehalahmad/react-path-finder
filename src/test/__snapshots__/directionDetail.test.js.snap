// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DirectionDetail /> direction detail is rendered 1`] = `ShallowWrapper {}`;

exports[`<DirectionDetail /> direction detail is rendered 2`] = `ShallowWrapper {}`;
